### PR TITLE
doc: add quick start example

### PR DESCRIPTION
**Pull request by ronag**

### CODE REVIEW

**Comment by ronag** [comment]
@dnlup @Ethan-Arrowood

**Review by dnlup — APPROVED** [review]
lgtm

**Review by Ethan-Arrowood — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -33,6 +33,28 @@ undici - dispatch x 47,319 ops/sec ±1.17% (263 runs sampled)
...
+
+```js
+const undici = require('undici')
+
+const {
+  statusCode,
+  headers,
+  trailers,
+  body
+} = await undici.request('http://localhost:3000')
```
Comment (mcollina) [inline README.md:46]: TLA will not work unless using esm. I would recommend switching this to ESM

**Review by mcollina — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**